app/controllers/settlements_controller.rb

```diff
@@ -0,0 +1,6 @@
+class SettlementsController < ApiController
+  def index
+    @municipality = Municipality.find_by(name:params[:municipality])
+    render json: @municipality.settlements.order(name: :asc).distinct if !params[:municipality].nil?
+  end
```

Comment:
I think the guard could be at top of the method, and in case there isn't a municipality param should return and error, something like:
```suggestion
  def index
    return some_error_handler unless params[:municipality]

    @municipality = Municipality.find_by(name:params[:municipality])
    render json: @municipality.settlements.order(name: :asc).distinct
  end
```

Reply:
Great!. Same as above.